### PR TITLE
Added Silverfort Solution

### PR DESCRIPTION
**Pull request by FrankGasparovic**
**Change(s):**
- Added **SolutionMetadata.json** to define metadata for the Silverfort solution in Microsoft Sentinel.
- Included **SilverfortWorkbook.json** to provide a detailed workbook template offering insights into Silverfort logs and events.
- Added **Certrifried.yaml**, **Log4Shell.yaml**, **NoPac_Breach.yaml**, and **User_Brute_Force.yaml** to create analytic rules for detecting specific incidents reported by Silverfort.
- Included **SilverfortITDROnPrem.json** to configure the Silverfort ITDR Admin Console data connector for Microsoft Sentinel.

**Reason for Change(s):**
- To integrate Silverfort’s identity threat detection and response capabilities within Microsoft Sentinel, providing enhanced security insights, analytics, and incident detection.
- This integration allows users to monitor, analyze, and respond to Silverfort-related events directly within their Sentinel workspace, leveraging advanced detection rules and a custom data connector.

**Version Updated:**
- Yes
- The version for Detections/Analytic Rule templates has been updated to 1.0.0 for the new rules: Certrifried, Log4Shell, NoPac_Breach, and User_Brute_Force.

**Testing Completed:**
- Yes
- All components were tested in a Microsoft Sentinel environment to ensure they work as expected. The analytic rules were validated to trigger correctly, the workbook displayed the intended data, and the data connector ingested logs without errors.

**Checked that the validations are passing and have addressed any issues that are present:**
- Yes
- All necessary validations were run locally, and any identified issues were resolved prior to submission.

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @FrankGasparovic, Please add input file which should be in Data folder. It is needed to package the solution. 
You can refer to this Solution for more clarification - https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Agari

Also add workbook metadata to this file 
https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json

Analytic rules are missing with entity mappings, techniques and tactics. 

Please do share sample data to test the content of the solution.

**Comment by v-prasadboke** [comment]
Also I have one doubt. 

Is this Solution Microsoft supported. We haven't received any instructions for the same. 
If it is not please correct the support details in solution metadata and publisher id should be different

**Comment by FrankGasparovic** [comment]
@v-prasadboke The PR has been updated based on your feedback. Please let me know if there is anything else needed.

File: Solutions/Silverfort/Data Connectors/SilverfortAma.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
+    "id": "SilverfortAma",
+    "title": "Silverfort Admin Console",
+    "publisher": "Silverfort",
+    "descriptionMarkdown": "The [Silverfort](https://silverfort.com) ITDR Admin Console connector solution allows ingestion of Silverfort events and logging into Microsoft Sentinel.\n Silverfort provides syslog based events and logging using Common Event Format (CEF). By forwarding your Silverfort ITDR Admin Console CEF data into Microsoft Sentinel, you can take advantage of Sentinels's search & correlation, alerting, and threat intelligence enrichment on Silverfort data. \n Please contact Silverfort or consult the Silverfort documentation for more information.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "DATATYPE_NAME",
```
Comment (v-prasadboke) [inline Solutions/Silverfort/Data Connectors/SilverfortAma.json:9]: Please replace DATA_TYPE name with data type name. 
You can refer to this Connector for more clarification 
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OneIdentity/Data%20Connectors/OneIdentity.JSON

File: Logos/silverfort-logo.svg
Comment (v-prasadboke) [inline Logos/silverfort-logo.svg:1]: Logo size should be below 5kb

File: Solutions/Silverfort/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "silverfort5120441444412",
+	"offerId": "silverfort",
```
Comment (v-prasadboke) [inline Solutions/Silverfort/SolutionMetadata.json:3]: Please include sentinel keyword in offer id 

image

File: Solutions/Silverfort/Data Connectors/SilverfortAma.json (hunk 1 above)
Comment (FrankGasparovic) [reply Solutions/Silverfort/Data Connectors/SilverfortAma.json]: Resolved

File: Logos/silverfort-logo.svg
Comment (FrankGasparovic) [reply Logos/silverfort-logo.svg:1]: Resolved

File: Solutions/Silverfort/SolutionMetadata.json (hunk 2 above)
Comment (FrankGasparovic) [reply Solutions/Silverfort/SolutionMetadata.json]: Resolved

**Comment by v-prasadboke** [comment]
Hello @FrankGasparovic, Thanks for committing the required changes. We will get back to you by 28 August, 2024.

**Comment by FrankGasparovic** [comment]
Hi @v-prasadboke,

I wanted to check on on the current status of the PR?

**Comment by v-prasadboke** [comment]
Hello @FrankGasparovic, Please update the branch from master. Also if possible can you please try to provide me write access to your branch. Need to push some required commits

**Comment by FrankGasparovic** [comment]
@v-prasadboke I have pulled in the latest upstream changes. I unfortunately can not give you write permissions on our forked repository. Can you please provide me the files you need to add or replace? Feel free to email me at frank.gasparovic@silverfort.com with the zip.

**Comment by v-prasadboke** [comment]
Hello @FrankGasparovic, We want you package the solution using V3 tool. Also add release notes to the solution. 
V3 tool - https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.md

Release notes reference - https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/IPinfo/ReleaseNotes.md
Release notes should be same as referred above

**Comment by FrankGasparovic** [comment]
@v-prasadboke The solution has been packaged with v3 and we have added release notes

**Comment by v-prasadboke** [comment]
Hello @FrankGasparovic, Can you please try to resolve failing validation errors

**Comment by FrankGasparovic** [comment]
@v-prasadboke All checks have been resolved

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**